src/bionics.cpp

```diff
@@ -2106,6 +2125,12 @@ bool Character::can_uninstall_bionic( const bionic_id &b_id, player &installer,
                disp_name( true ), disp_name() );
         return false;
     }
+    
+    if( b_id == bio_armor_arms || b_id == bio_armor_legs || b_id == bio_armor_head || b_id == bio_armor_torso ) {
```

Comment:
PLease try to avoid hardcoding more CBMs. Can you instead make it a json property?